### PR TITLE
[kesch] Remove PKG_CONFIG from netCDF-Fortran-4.4.4-CrayCCE-17.06

### PR DESCRIPTION
**Pull request by victorusu**

### CODE REVIEW

**Comment by victorusu** [comment]
I am not adding the CI here, since it was already tested by the users.

File: easybuild/easyconfigs/n/netCDF-Fortran/netCDF-Fortran-4.4.4-CrayCCE-17.06.eb [hunk 1]
```diff
@@ -4,10 +4,11 @@ easyblock = 'ConfigureMake'
 
 name = 'netCDF-Fortran'
 version = "4.4.4"
+versionsuffix = '-no-pkg-config'
```
Comment (gppezzi) [inline easybuild/easyconfigs/n/netCDF-Fortran/netCDF-Fortran-4.4.4-CrayCCE-17.06.eb]: please remove the versionsuffix, this was only useful for the testing and we're not planning to keep two versions of this modules.

Comment (victorusu) [reply easybuild/easyconfigs/n/netCDF-Fortran/netCDF-Fortran-4.4.4-CrayCCE-17.06.eb]: done! 👍

**Comment by gppezzi** [comment]
@victorusu This isn't a long build so I don't see why you would skip the CI.

**Comment by gppezzi** [comment]
retest this please

**Comment by gppezzi** [comment]
CI is not ready for testing stuff based on the Cray modules because of MCH's custom naming scheme  (now that's a valid reason for skipping the CI :)

**Pull request merged**